src/config.js

```diff
@@ -13,7 +13,9 @@ var OAUTH_API_KEY = 'MyM7bhB25wdkNKI2OCa6137E1u8';
 
 var TC_ENDPOINT_DEV = 'https://glib-mock.herokuapp.com/';
 var TC_ENDPOINT_PROD = 'https://glib-prod.herokuapp.com/';
-var EXTENSION_ID = 'kbdpmophclfhglceikdgbcoambjjgkgb';
+var EXTENSION_ID = chrome.runtime.id;
```

Comment:
These are the known development and production chrome extension ids...needed for TC auth callbacks.